bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCreateTable.java

```diff
@@ -113,4 +117,58 @@ public void testTableNames() throws IOException {
       }
     }
   }
+  
+  @Test
```

Comment:
As per the offline comments, please add @Category(KnownGap.class) since this fails against production.